src/Core/Addon.php

```diff
@@ -185,6 +186,25 @@ public static function loadHooks()
 		dba::close($r);
 	}
 
+	/**
+	 * @brief Forks a hook.
+	 *
+	 * Use this function when you want to fork a hook via the worker.
+	 *
+	 * @param string $name of the hook to call
+	 * @param string|array $data to transmit to the callback handler
+	 */
+	public static function ForkHooks($priority, $name, $data = null)
```

Comment:
Guess what? Right idea, wrong capitalization: it should be `forkHooks`.

Reply:
One day I will learn ...

Reply:
`ClassName`
`methodName`
`$variable_name`

Reply:
Done.